code/game/machinery/computer/card.dm

```diff
@@ -413,7 +413,11 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 							updateUsrDialog()
 							break
 					if(!jobdatum)
-						to_chat(usr, "<span class='error'>No log exists for this job.</span>")
+						to_chat(usr, "<span class='warning'>No log exists for this job.</span>")
+						updateUsrDialog()
+						return
+					if(!isnull(modify.registered_age) && modify.registered_age < jobdatum.minimal_character_age)
+						to_chat(usr, "<span class='warning'>This individual is too young to hold that Job, per Nanotrasen guidelines. Job assignment aborted!</span>")
```

Comment:
I'd rather this would alert the user and ask them to confirm to do it in spite of their age limit, so you don't *need* to go and falsify records to change the job if need be.

Reply:
That'd certainly be the more passive approach, and if you look that is genuinely how I implemented it earlier.

However, I think having this instead kinda plays well into the idea of the HoP being unnecessarily bureaucratic, actually providing a gameplay element that encourages that. Plus this would be pretty much the sole and only time in the entire video game where one's IC age has any mechanical effect. A mechanical effect that you can override by altering an integer record.

That's why I changed it from being like as you've suggested towards this.